documentation/book/con-cluster-operator-namespaces.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// assembly-operators-cluster-operator.adoc
+
+[id='con-cluster-operator-namespaces-{context}']
+= Watching multiple namespaces
+
+The Cluster Operator is able to watch for the `Kafka`, `KafkaConnect` and `KafkaConnectS2I` resources in multiple namespaces.
+It does not have to be installed in every single namespace separately.
+The only reason to run multiple Cluster Operators in a single cluster is to use different versions of {ProductName}.
+
+// TODO: Link to env var definition
+
+The `STRIMZI_NAMESPACE` environment variable can be used to configure a single operator instance to operate in multiple namespaces.
+For each namespace given, the operator will watch for the `Kafka`, `KafkaConnect` and `KafkaConnectS2I` and perform periodic reconciliation.
```

Comment:
for the `Kafka`, `KafkaConnect` and `KafkaConnectS2I`  > for `Kafka`, `KafkaConnect`, and `KafkaConnectS2I`